src/docs/sdk/performance/index.mdx

```diff
@@ -32,6 +34,41 @@ Optionally, the `tracesSampler` callback can also return a boolean to force a sa
 
 See more about how sampling should be performed below.
 
+### `tracePropagationTargets`
+
+Sentry SDKs propagate trace information via headers of outgoing Http requests to downstream SDKs (see [`sentry-trace`](#header-sentry-trace) and [`baggage`](/sdk/performance/dynamic-sampling-context/#baggage) for more details).
+The `tracePropagationTargets` option gives users a mechanism of controlling to which outgoing Http requests these headers should be attached.
+For example, users can specify this property to keep trace propagation within their infrastructure, thereby stopping it from being sent to third party services.
+
+This option takes an array of strings or regular expressions to match against the URLs of outgoing requests.
+If this option is specified by users, SDKs may only add the trace headers to a request if the the request's URL matches (contains) at least one of the items from the array.
+
+#### Example
+
+```JS
+// Entries can be strings or RegEx
+tracePropagationTargets: ['localhost', /^\// ,/myApi.com\/v[2-4]/]
+
+URLs matching: 'localhost:8443/api/users', '/api/envelopes', 'myApi.com/v2/projects'
+URLs not matching: 'someHost.com/data', 'myApi.com/v1/projects'
+```
+
+#### Default Behaviour
+
+If the `tracePropagationTargets` option is not specified by users, SDKs should fall back to their default behaviour for trace propagation.
+This default behaviour is not standardized and can vary across SDKs.
+For instance, Server-side SDKs may propagate trace headers to all outgoing requests by default while client-side SDKs (e.g. Javascript) can fall back to a default array of allowed domains (e.g. `localhost` and `/^\//` in JavaScript).
+Choosing a sensible default behaviour is up to the SDK maintainers, however, special limitations, such as CORS issues for frontend web SDKs must be considered.
+SDKs that previously supported the `tracingOrigins` property (see deprecation notice below) can continue using the already established default behaviour for `tracePropagationTargets`
+
+<Alert level="info" title="Deprecation of tracingOrigins">
+
+This Option replaces the non-standardized `tracingOrigins` option which was previously used in some SDKs.
+SDKs that support `tracingOrigins` are encouraged to deprecate and and eventually remove `tracingOrigins` in favour `tracePropagationTargets`.
+In case both options are specified by users, SDKs should only rely on the `tracePropagationTargets` array.
```

Comment:
```suggestion
This option replaces the non-standardized `tracingOrigins` option which was previously used in some SDKs. SDKs that support `tracingOrigins` are encouraged to deprecate and and eventually remove `tracingOrigins` in favour `tracePropagationTargets`. In case both options are specified by users, SDKs should only rely on the `tracePropagationTargets` array.
```

(No change other than pulling the extra newlines.)

Reply:
What I was going for with the newlines was easier readability and for instance less clutter when reviewers would e.g. make wording suggestions in one sentence. 
I think from a MD rendering perspective, having one or no newline after each sentence shouldn't make any difference (as long as it's just one newline obv.). 
Came across this in various PRs of our docs repos and I thought it makes sense. But admittedly, I didn't look if we have a convention for this or not.

Reply:
Ok, did some googling and it turns out that this apparently is a debated topic (example [one](https://github.com/github/choosealicense.com/issues/505), [two](https://twitter.com/jakevdp/status/796817091684769792), + a ton more). Another advantage is less conflicts/better diffing in git.

Reply:
It's interesting - this is the first time I've ever seen it done. Normally people do hard wrapping, which is _awful_ for diffing, and (though I can't find it anywhere now among our 17 docs style guides) I could swear that at one point we actually had "one line per paragraph" as part of our docs guidelines.

I do see the point about diffing. At the same time, I find it significantly harder to read, both from a shape perspective (my eyes have to move a different amount for every line) and from a semantic perspective (the newline makes my brain pause for a second, so the text doesn't flow). It also means one has to think about which newlines actually are paragraph breaks and which aren't, especially given the line padding the GH does (which makes it look a little like each line is a new paragraph).

It's true that GH didn't do a great job of this in the suggestion above, but I generally find that the brighter highlighting that it/VSCode/Fork/etc normally do is enough:

![image](https://user-images.githubusercontent.com/14812505/179853690-a1ad615c-10be-4f30-83fa-e615b4e23592.png)

So IDK. Unless/until we reach a point where we decide that's what we do, everywhere (and we all adjust to reading that way), my preference would be to leave them out. But your PR, your call. 🙂

Reply:
Ok, let's go with consistency (changed it in [bd4a295](https://github.com/getsentry/develop/pull/648/commits/bd4a295e5f155d08c19f09aecd9ff18d2f82a6e3)). 

I think it might be worth having this discussion in the future (maybe if we ever get a #docsengineer 👀 ). I personally find the one-sentence-lines easier to read because I'm not a fan of eternally long lines (also with auto wrap and stuff) but I agree that consistency is more important. (Maybe it's just PTSD from my LaTeX days at university where this was even more of a problem)